test/include/execution/sql_test.h

```diff
@@ -51,17 +51,11 @@ class SqlBasedTest : public TplTest {
   common::ManagedPointer<storage::BlockStore> BlockStore() { return block_store_; }
 
   std::unique_ptr<exec::ExecutionContext> MakeExecCtx(exec::OutputCallback *callback = nullptr,
-                                                      const planner::OutputSchema *schema = nullptr,
-                                                      bool force_serial = false) {
```

Comment:
ah, a default parameter. that's why it escaped noticed. :eyes: 

(fyi we try to avoid adding default parameters too)